src/components/CouponDetails/index.jsx

```diff
@@ -463,7 +470,10 @@ class CouponDetails extends React.Component {
 
   handleVisibilitySelect(newValue) {
     const { tableColumns, visibilityToggle } = this.state;
-    const value = newValue || visibilityToggle;
+    // Paragon InputSelect will use the `label` if the value isn't defined
+    // this will intentionally keep the value set as undefined so that qs.stringify
+    // won't send along a value we didn't set to the API.
+    const value = newValue === 'Both' ? undefined : newValue || visibilityToggle;
```

Comment:
I prefer `null` to `undefined` when it's an intentionally falsey value, but that's a to total nitpick.

Reply:
Would _also_ prefer it but the qs package will serialize null params but will drop undefined ones which is what we want here